api.bs

```diff
@@ -967,7 +967,11 @@ updated to refer to the [=privacy budget store=] as well.
 ### Privacy Budget Store ### {#s-privacy-budget-store}
 
 The <dfn>privacy budget store</dfn> is a [=map=] whose keys are
-[=privacy budget keys=] and whose values are [=floats=].
+[=privacy budget keys=] and whose values are [=32-bit unsigned integers=].
+These [=32-bit unsigned integer|integers=] store a value
+in units [=microepsilons=].
+A <dfn>microepsilon</dfn> is one-millionth of the unitary epsilon value
+used in differential privacy [[DP]].
```

Comment:
```suggestion
used in differential privacy [[DP]].

<p class=note>  The choice of a 32 bit integer restricts the setting of epsilon to be less than 4295.
This should be more than sufficient for implementations.
```

Reply:
should that say "4095" instead?

Reply:
I don't think so, it is the result of `floor((2^32 - 1) / 1_000_000)`

Reply:
Oh got it; my bad for dividing by 1_048_576 instead